python/tests/test_safari.py

```diff
@@ -4,7 +4,7 @@
 class TestSafari(object):
 
     def test_defaults(self):
-        sauce = SauceOptions('safari')
+        sauce = SauceOptions('safari', platformName='macOS 10.14')
```

Comment:
Are we going to require Safari versions? This seems inconsistent with the other browser cases (eg Microsoft Edge)?

Reply:
Yeah, Safari is special because the default ("Windows 10") doesn't work. We can either hard code what is currently "latest" and handle potential issues with order of operations or we can require a user to specify. I'm recommending we go with the latter for now. We either need a way to identify the places in the code that can become stale (e.g. mark them with a comment `Maintenance Required`), or create a job that will automatically generate the enum/properties/yaml/whatever file that gets used to determine correct values.

Reply:
Ah yes, I recall you wanting to do the scripting approach fo generate values. We can do this approach for now.